epochtalk/epochtalk
# Epoch themes

Test procedure:

- Angular has been updated to 1.4.4 and foundation has been removed. 
- Delete bower_components folder then run bower install.
- Remove foundation.scss, /foundation, and normalize.scss from your scss folder in app.
- Start the server
- Verify that the desktop/mobile theme on all pages looks ok
